lib/cli/commands/subscribealerts.ts

```diff
@@ -0,0 +1,72 @@
+import { ServiceError, status } from 'grpc';
+import { Arguments, Argv } from 'yargs';
+import { XudClient } from '../../proto/xudrpc_grpc_pb';
+import * as xudrpc from '../../proto/xudrpc_pb';
+import { setTimeoutPromise } from '../../utils/utils';
+import { loadXudClient } from '../command';
+import { AlertType } from '../../constants/enums';
+
+export const command = 'subscribealerts [--pretty]';
+
+export const describe = 'subscribe alerts such as low balance';
+
+export const builder = (argv: Argv) => argv
+    .option('pretty', {
+      type: 'boolean',
+    })
+    .example('$0 subscribealerts', 'prints alert payload in a JSON structure')
+    .example('$0 subscribealerts --pretty', 'prints alert message only');
+
+export const handler = async (argv: Arguments) => {
+  await ensureConnection(argv, true);
+};
+
+let client: XudClient;
+
+const ensureConnection = async (argv: Arguments, printError?: boolean) => {
+  if (!client) {
+    client = await loadXudClient(argv);
+  }
+  client.waitForReady(Date.now() + 3000, (error: Error | null) => {
+    if (error) {
+      if (error.message === 'Failed to connect before the deadline') {
+        console.error(`could not connect to xud at ${argv.rpchost}:${argv.rpcport}, is xud running?`);
+        process.exit(1);
+      }
+
+      if (printError) console.error(`${error.name}: ${error.message}`);
+      setTimeout(ensureConnection.bind(undefined, argv, printError), 3000);
+    } else {
+      console.log('Successfully connected, subscribing for alerts');
+      subscribeAlerts(argv);
+    }
+  });
+};
+
+const subscribeAlerts = (argv: Arguments<any>) => {
+  const request = new xudrpc.SubscribeAlertsRequest();
+  const alertsSubscription = client.subscribeAlerts(request);
+
+  alertsSubscription.on('data', (alert: xudrpc.Alert) => {
+    if (argv.pretty) {
+      console.log(`${AlertType[alert.getType()]}: ${alert.getMessage()}`);
+    } else {
+      console.log(JSON.stringify(alert, undefined, 2));
+    }
+  });
+  alertsSubscription.on('end', reconnect.bind(undefined, argv));
+  alertsSubscription.on('error', async (err: ServiceError) => {
+    if (err.code === status.UNIMPLEMENTED) {
```

Comment:
It might also be nice to break this into a utils function `onStreamError` so that we reuse the code rather than duplicate it from streamorders.ts.